### ISSUE TITLE
Display HTTP method in the sidebar in "read" mode

### ISSUE THREAD

**Issue by NoelOConnell** [mrin9/RapiDoc#210]
Hi,

Is it possible to display the HTTP method next to the endpoint description.

On the homepage, you can see this in one of the screenshots.
https://mrin9.github.io/RapiDoc/images/rapi-mode-read.png

However, if you look at the examples the HTTP method is not included. Only the description is listed.

![image](https://user-images.githubusercontent.com/966495/81590719-2c4e9d00-93b3-11ea-94f4-33b7438c37e9.png)

**Comment by mrin9** [comment]
that homepage screen shot you referenced in the link is quite old, I need to fix that

but to answer your question, we have been through several iterations of user feedbacks (_our user-group constitute of API-Developers, Tech Writers, API-Customers_)

from those surveys, It was clear to us that a single rendering style wont be able to satisfy various user-groups, therefore we introduced  two rendering styles - 
- `view` geared for developers (or those who are more interested in API implementation details)
- `read` geared for tech readers, customer facing API guides

`read` mode do try to go easy on tech-details (so we dont show the `paths` or the `HTTP method` on the nav bar) however when you select you get to see it but we are subtle about it

In a read mode the expectation is the  `path-summary` should contain the language that is self explanatory as to what operation it is performing, therefore we provide more room for the text and hide technical details. also the `HTTP method` makes more sense when they are displayed along with the `API path`, so if thats the thing you are after i would suggest to use `render-style='view'`

**Comment by NoelOConnell** [comment]
Thanks for the explaination.

I was looking at a few other documentation viewers like Redoc, Spotlight and Readme in the past few days. They all have a sidebar with HTTP method name and text.

So when I saw the sidebar in RapiDoc I subconsiciously assumed it would behave the same when I looked at the screenshot on the homepage.

Using `render-style="view"` and `layout="row"` works for me.

RapiDoc is really great. It's been so easy to use and config. 
Thanks for your help.
